src/base/static/components/atoms/typography.js

```diff
@@ -108,6 +109,20 @@ Header6.propTypes = {
   classes: PropTypes.string,
 };
 
+// TODO: Other label types.
+const RegularLabel = styled("label")(
```

Comment:
I see you point about differentiating between *Label and *Text typography atoms. But what's wrong with calling these *Label atoms, instead of *Title atoms?

Also, is `Regular` referring to the size, or something else? What's wrong with calling this `MediumLabel` (or `MediumTitle`)?

Reply:
I guess for me a `*Title` component would signal the title of a section of a document, collecting the purpose of text to follow in a short phrase. 

The semantic purpose of a `*Label`, though, is to attach meaning to a graphical element. The [MDN docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label) say that a `<label>` element 

> represents a caption for an item in a user interface.

So semantically I think the `*Label` components are appropriate for something like a map legend. Plus I think there is an i18n benefit to using semantic markup as much as possible.

Reply:
I have mixed feelings about `RegularLabel`-- I noticed the Airbnb design language uses the term `regular`, which is why I used it here. One advantage of using that naming convention would be to signal the default choice-- i.e. if you don't have a specific reason for using a `SmallLabel`, just default to `RegularLabel`.

Reply:
Ah, I didn't see this was a `label` element, instead of a `div`. So yeah, I think *Label makes sense here.

>One advantage of using that naming convention would be to signal the default choice-- e.g. if you don't have a specific reason for using a SmallLabel, say, just default to RegularLabel

I like that idea :+1: Maybe we can go over our Design Language strategy tomorrow to make sure we are on the same page about these things.